src/main/java/com/skankhunt220/api/controller/UserController.java

```diff
@@ -1,72 +1,73 @@
 package com.skankhunt220.api.controller;
 
-import java.io.IOException;
-import java.io.PrintWriter;
-import java.util.stream.Collectors;
-
-import javax.servlet.ServletException;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
+import java.util.Optional;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.gson.Gson;
 import com.skankhunt220.entity.User;
-import com.skankhunt220.service.UserService;
+import com.skankhunt220.repository.UserRepository;
 
 @Controller
-@RequestMapping(value = "/user", produces = {"application/json"})
+@RestController
+@RequestMapping("/api/users")
 public class UserController {
-	private static final UserService userService = new UserService();
-	private static final ObjectMapper mapper = new ObjectMapper();
-	
-	@ResponseBody 
-	private void sendResponse(String message, HttpServletResponse response) throws IOException {
-		response.setContentType("application/json");
-		try (PrintWriter out = response.getWriter()) {
-			out.print(message);
-		}
-	}
-	private User getUserInfo(HttpServletRequest request) throws IOException {
+	@Autowired
+	private UserRepository userRepository;
+
+	@PostMapping("/")
+	public ResponseEntity<User> createUser(@RequestBody User user) {
```

Comment:
The class named a `UserController`, so don`t need add to method names **user** prefix